src/services/ui/src/pages/actions/ToggleRaiResponseWithdraw.tsx

```diff
@@ -62,7 +87,19 @@ export const ToggleRaiResponseWithdraw = ({
       )}
       <div className="flex gap-2">
         <Button onClick={() => mutate()}>Submit</Button>
-        <Button onClick={() => setCancelModalOpen(true)} variant="outline">
+        <Button
+          onClick={() => {
+            setModalContent({
+              header: "Stop form submission?",
+              body: "All information you've entered on this form will be lost if you leave this page.",
+              acceptButtonText: "Yes, leave form",
+              cancelButtonText: "Return to form",
+            });
+            setModalOnAccept(() => acceptAction);
```

Comment:
I haven't tested yet, but this looks like it needs to be updated to:

```js
setModalOnAccept(acceptAction);
```

Reply:
This works as-intended. To properly test/confirm, change the `() => acceptAction` to `acceptAction` and try the Cancel button on this, or the other, forms.